<?xml version="1.0"?>
<launch>
    <arg name="distance_laser" default="0.4"/>
    <arg name="speed_factor" default="1.0"/>
    <arg name="forward_speed" default="0.2"/>
    <arg name="backward_speed" default="-0.2"/>
    <arg name="rotation_speed" default="0.5"/>

    <node pkg="my_robot_control" exec="my_robot_wallfollower_exec" name="wall_follower_node" output="screen">
        <param name="distance_laser" value="$(var distance_laser)"/>
        <param name="speed_factor" value="$(var speed_factor)"/>
        <param name="forward_speed" value="$(var forward_speed)"/>
        <param name="backward_speed" value="$(var backward_speed)"/>
        <param name="rotation_speed" value="$(var rotation_speed)"/>
    </node>
</launch>